recipes/optional-lite/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, CMake, tools
```

Comment:
According to review of the other nonstd libs recipes it's better to remove 'CMake' from here.

Reply:
It looks like we should add a Python linter to the CI checks